dev-docs/bidders/risexchange.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: bidder
+title: RiseXChange
+description: Prebid RiseXChange Bidder Adapter
+multiformat_supported: will-bid-on-any
+pbjs: true
+biddercode: risexchange
+media_types: banner, video
+schain_supported: true
+coppa_supported: true
+pbs: true
+tcfeu_supported: true
+usp_supported: true
+floors_supported: true
+userIds: all
+gvl_id: 1043 (rise)
```

Comment:
```suggestion
gvl_id: 1043
```

You don't need to specify the biddercode here as this is your own ID as far as I can tell :smiley: